Mathlib/Algebra/Group/Ext.lean

```diff
@@ -125,7 +125,7 @@ theorem DivInvMonoid.ext {M : Type*} ⦃m₁ m₂ : DivInvMonoid M⦄
     ext a b
     exact @map_div' _ _
       (F := @MonoidHom _ _ (_) _) _ (id _) _
-      (@MonoidHom.instMonoidHomClass _ _ (_) _) f (congr_fun h_inv) a b
+      (@MonoidHom.instMonoidHomClass _ _ (_) _).toMulHomClass f (congr_fun h_inv) a b
```

Comment:
Ideally, all of Mathlib would avoid referencing `@Some.instName`.
But that is quasi-orthogonal to your current generalization effort.

Reply:
Looking at the history this has been there since the port. I've replaced it with inferInstance and will see what the bench looks like.

Reply:
This is very weird code that is dealing with two instances of the same typeclass at once; at one point the proof did not compile without spelling all of this out explicitly.